micrometer-core/src/main/java/io/micrometer/core/instrument/binder/mongodb/MongoMetricsCommandListener.java

```diff
@@ -61,9 +83,7 @@ public void commandFailed(CommandFailedEvent event) {
 
     private void timeCommand(CommandEvent event, String status, long elapsedTimeInNanoseconds) {
         timerBuilder
-                .tag("command", event.getCommandName())
-                .tag("cluster.id", event.getConnectionDescription().getConnectionId().getServerId().getClusterId().getValue())
-                .tag("server.address", event.getConnectionDescription().getServerAddress().toString())
+                .tags(tagsProvider.commandTags(event))
                 .tag("status", status)
```

Comment:
the `status` tag is provided by tagsProvider already, so probably we shouldn't need to duplicate it here

Reply:
True. I will update w/in the next 24 hrs.

Reply:
You could also change the timeCommand parameter list to drop the `status` one

Reply:
Ok, I finally got back around to updating this. Removing the `status` did clean up the code a bit. I originally left it in thinking that status is something that is core to the listener and *should* always be provided. However, that is rather presumptuous of me and the next consumer may not want status, may want to call it something else - this is the whole purpose of the tags provider :) 

I rebased and squashed as it was out of date. I was not going to squash but my keyboard nad muscle memory had a different plan.